source/_integrations/device_tracker.markdown

```diff
@@ -99,20 +99,28 @@ In the example above, `devicename` refers to the detected name of the device.  F
 | `track`         | [uses platform setting]       | If  `yes`/`on`/`true` then the device will be tracked. Otherwise its location and state will not update.                                                                                                                                                                                                                                        |
 | `consider_home` | [uses platform setting]       | Seconds to wait till marking someone as not home after not being seen. Allows you to override the global `consider_home` setting from the platform configuration on a per device level.                                                                                                                                                         |
```

Comment:
Side note: I think we should show somehow that these settings are only applicable for legacy platforms configured from YAML. This is going away as integrations are migrated to config entries.

The same applies for `known_devices.yaml`.